src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/automatic-remove/AutomaticRemoveView.tsx

```diff
@@ -26,6 +27,32 @@ export function AutomaticRemoveView(props: Props) {
   const { monthlySubscriptionUrl, yearlySubscriptionUrl, ...fixViewProps } =
     props;
 
+  // format subscription urls
```

Comment:
Since these two values are 99% similar, not sure if it's easier to do a partial or some helper to make sure they stay in sync.
```js
const addAttributions = (url) => modifyAttributionsForUrl(url, 
    {
      entrypoint: "monitor.mozilla.org-monitor-in-product-guided-upsell",
      form_type: "button",
    },
    {
      utm_source: "product",
      utm_medium: "monitor",
      utm_campaign: "guided-upsell",
    },
);

const monthlySubscriptionUrlWithAttributions = addAttributions(monthlySubscriptionUrl);
const yearlySubscriptionUrlWithAttributions = addAttributions(yearlySubscriptionUrl);
```